src/main/scala/nl.knaw.dans.easy.sword2/servlets/EasySword2Servlet.scala

```diff
@@ -26,6 +26,6 @@ class EasySword2Servlet(version: String) extends ScalatraServlet
 
   get("/") {
     contentType = "text/plain"
-    Ok(s"EASY SWORD2 is running v$version.").logResponse
+    logResponse(Ok(s"EASY SWORD2 is running v$version."))
```

Comment:
I think the response is logged twice now (after the upgrade of `dans-scala-lib`). Could you test if removing this whole `logResponse` still logs the response?

```suggestion
    Ok(s"EASY SWORD2 is running v$version.")
```